account/account.go

```diff
@@ -316,103 +423,329 @@ func (account *Account) WaitForTransactionReceipt(ctx context.Context, transacti
 	}
 }
 
+// AddInvokeTransaction generates an invoke transaction and adds it to the account's provider.
+//
+// Parameters:
+// ctx - the context.Context object for the transaction.
+// invokeTx - the invoke transaction to be added.
```

Comment:
The format is different here
```suggestion
// - ctx: the context.Context object for the transaction.
// - invokeTx: the invoke transaction to be added.
```